crates/ink/codegen/src/generator/mod.rs

```diff
@@ -70,3 +73,32 @@ pub use self::{
     storage_item::StorageItem,
     trait_def::TraitDefinition,
 };
+
+/// Check if the the conditional compilation expression
+/// for any `cfg` attributes is satisfied.
+///
+/// # Example
+/// If we have an annotated item (e.g. message)
+/// ```ignore
+/// #[ink(message)]
+/// #[cfg(feature = "foo")]
+/// #[cfg(feature = "baz")]
+/// pub fn get(&self) -> String {
+///     self.value.clone()
+/// }
+/// ```
+/// The function would would iterate over the `cfg` attributes
+/// and return `true` if any of the feature flags are set
+pub fn is_conditionally_excluded(attrs: &[Attribute]) -> bool {
```

Comment:
I would change this to the positive e.g. `any_conditional_cfg` since the most common usage is negated e.g. `!is_conditionally_excluded`. Easier to read and understand.

Reply:
Okay maybe a better name is something like `all_cfg_predicates_true`, since as I understand it:
```
#[cfg(feature = "foo")]
#[cfg(feature = "baz")]
```
is the equivalent of:
```
#[cfg(all(feature = "foo", feature = "bar"))]
```